Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -407,6 +408,30 @@ - (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
   callback(@[formattedScheduledLocalNotifications]);
 }
 
+RCT_EXPORT_METHOD(removeAllDeliveredNotifications)
+{
+  if ([UNUserNotificationCenter class]) {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center removeAllDeliveredNotifications];
+  }
+}
+
+RCT_EXPORT_METHOD(removeDeliveredNotifications:(NSArray<NSString *> *)identifiers)
+{
+  if ([UNUserNotificationCenter class]) {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center removeDeliveredNotificationsWithIdentifiers:identifiers];
+  }
+}
+
+RCT_EXPORT_METHOD(getDeliveredNotifications:(RCTResponseSenderBlock)callback)
+{
+  if ([UNUserNotificationCenter class]) {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center getDeliveredNotificationsWithCompletionHandler:callback];
```

Comment:
Yes, i get this error `RCTJSONStringify() encountered the following error: Invalid type in JSON write (UNNotification)`... Can you help me with that? How can i do that?